tests/test_correctness.py

```diff
@@ -41,16 +41,16 @@ def test_correctness_filepath(cc):
     assert filepath == f'{cc.correctness_results_folder}/1.csv'
 
 
-def test_has_differences(cc):
+def check_for_mismatches(cc):
     first_df = pd.DataFrame(columns=['col'])
     second_df = pd.DataFrame(columns=['col'])
     foobar = {'foo': ['Hello'], 'bar': ['World']}
 
-    assert not cc.has_differences(first_df, second_df)
-    assert cc.has_differences(first_df, pd.DataFrame(foobar))
-    assert cc.has_differences(pd.DataFrame(foobar), second_df)
-    assert not cc.has_differences(pd.DataFrame(foobar), pd.DataFrame(foobar))
-    assert cc.has_differences(pd.DataFrame({'foobar': ['Hello']}),
+    assert not cc.check_for_mismatches(first_df, second_df)
```

Comment:
how about we add some tests for:
- mismatching on digits:
-- 0.001 difference (but bigger than 1e-12)
-- 1 difference, but smaller than 1e-12,
-- etc
- rows that are unordered
- different column orderings
- nan values
- empty rows
- empty values
- sets of rows where the rows do not match but if you would order the values they are all there

Reply:
Yes, good idea.